.gitlab-ci.yml

```diff
@@ -1,13 +1,13 @@
 # This file is locked for GovCMS SaaS.
 include:
 
-  # This file allows some control of GitLab testing phases.
-  - local: '/.gitlab-ci-defaults.yml'
+  # Starting inputs for the GitLab CI configuration.
+  - local: '/.gitlab-ci-inputs.yml'
```

Comment:
`inputs`? what does it mean?

Reply:
inputs to an operation.

Reply:
no, i understand what the word `input` means :)) I was asking from the consumer developer POV. Previsously, it was suffixed with `-defaults` and it was clear (kinda). Now it is not clear who and when can change those.

Reply:
Ah gotcha, we did a bit of a poll internally on the name and `inputs` won unanimously. I agree "defaults" means something but it might mean a little too much - we want the meaning to be more like "switches" or "flags".

Reply:
Looks like the polling was on the wrong context. :) This file is by nature is a config, which implies "switches" or "flags" (which are all inputs). Perhaps the poll should be on "who and when using this file in the GoVCMS ecosystem". Is it for GovCMS setup script, helpdesk, agency tech leads, agency devs? Is it for daily usage or once-off setup?

Reply:
It will be a touch point for a lot of people off diverse experience. Clients and helpdesk. People debugging.  A team wants to have an additional variable in a script without hard-forking the entire CI flow. Developers deciding to change their lint from "soft fail" to "hard fail". Non-technical users needing to unblock an important deployment that is blocked because developers  made linting hard fail, but haven't resolved the linting issues. Potentially AWX processes that can inject a different input into the CI, but that's just architecturally feasible, not really something we're doing immediately.

Lots of considerations. Hence I think inputs works because it has a technical meaning that limits wrong assumptions by lay people.

Reply:
No discounting wrong context for the decision, but it will undergo more review anyway so this is a way to road test it.

Reply:
Thank you for explaining. It would help to summarise all of the above into a comment and add above that line.
I'm leaving this one alone now :) Thanks